arango/client.py

```diff
@@ -64,6 +71,7 @@ def __init__(
         serializer: Callable[..., str] = lambda x: dumps(x),
         deserializer: Callable[[str], Any] = lambda x: loads(x),
         verify_override: Union[bool, str, None] = None,
+        request_timeout: Any = -1,
```

Comment:
Instead of introducing`-1` I think it might be better to use `60` as default to align with https://github.com/ArangoDB-Community/python-arango/blob/main/arango/http.py#L69

Reply:
@joowani, thanks for your review. We avoided giving the parameter a real default value (i.e. `60`) because of cases where a caller would first initialize an http client and configure a request timeout there already (let's say `200`) and then pass that object to the constructor of the `ArangoClient`. In this case, the configured request timeout on the custom http client (`200`) will be overwritten by our default value (`60`).

Reply:
I see. What about `None`? I would like to avoid introducing a new magic number `-1` to denote a behavior when we've already been using `None` everywhere in the library.

Reply:
I understand your concern with using `-1`. I have changed the default parameter value to `60` and updated to constructor to overwrite request timeout only if `http_client = None`. This way, it will not overwrite the request timeout on an existing `http_client`.